### PR TITLE
Create PRATYUSHPAHARI.json

### PR DESCRIPTION
**Pull request by PRATYUSHPAHARI**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3099"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by PRATYUSHPAHARI** [comment]
> Thanks! 🎉

When my linkfree account will be created ??

**Comment by Panquesito7** [comment]
> > Thanks! 🎉
> 
> When my linkfree account will be created ??

Your PR will be merged once another maintainer approves it.

**Pull request merged**
